### PR TITLE
Show hide columns

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

File: src/components/layer-table/layer-table.tsx [hunk 1]
```diff
@@ -598,11 +619,53 @@ export class LayerTable {
...
+            ) :
             this._getAction(cur.icon, cur.label, cur.func, cur.disabled)
         )
       }
       return prev;
-    }, [])
+    }, []);
+  }
+
+  _getFieldlist(): VNode {
```
Comment (MikeTschudi) [inline src/components/layer-table/layer-table.tsx:640]: Doc

File: src/components/layer-table/layer-table.tsx [hunk 2]
```diff
@@ -1183,6 +1278,50 @@ export class LayerTable {
...
+
+  /**
+   * Toggle show/hide dropdown
+   */
+  protected _toggleShowHide(): void {
+    this._showHideOpen = !this._showHideOpen;
+  }
+
+  /**
+   * Open show/hide dropdown
```
Comment (MikeTschudi) [inline src/components/layer-table/layer-table.tsx:1301]: "Open"?

**Review by MikeTschudi — APPROVED** [review]

File: src/components/layer-table/layer-table.tsx (hunk 2 above)
Comment (jmhauck) [reply src/components/layer-table/layer-table.tsx:1301]: No...this is a bool prop

**Comment by jmhauck** [comment]
#350

**Pull request merged**